### PR TITLE
Fix Android target

### PR DESCRIPTION
**Pull request by toledoal**
This PR fixes the Android target issues. This were happening because position:'absolute' cannot be use with display:'none' in Android. This PR also fixes the outline error on the text input. This error also was reported in iOS.

### CODE REVIEW

File: example/app.tsx [hunk 1]
```diff
@@ -2,14 +2,17 @@
 
 import React, { ReactElement } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-// import { SmartPrice } from './src/';
+// import { SmartPrice } from './src';
```
Comment (shubhagupta10) [inline example/app.tsx:5]: remove the comment?

File: index.tsx [hunk 2]
```diff
@@ -40,30 +47,54 @@ export const SmartPrice: FunctionComponent<ISmartPriceProps> = ({
   const slideScreen = () => {
     if (componentRef.current) {
       componentRef.current.measure((_fx, _fy, _width, _height, px, py) => {
-        setPosition({ left: -px, top: -py });
+        if (px && py) {
+          setPosition({ left: -px, top: -py });
+        }
       });
       setIsOpen(true);
+      setAndroidWidth(modalSize().width);
```
Comment (shubhagupta10) [inline index.tsx:55]: it will be called for all devices?

Comment (toledoal) [reply index.tsx:55]: good point, it should be called only on Platform.OS === 'android'

File: src/modal/smartprice-modal.tsx [hunk 3]
```diff
@@ -310,6 +310,12 @@ export const SmartpriceModal: FunctionComponent<ISmartpriceModalProps> = ({
   const activityIndicatorStyle: ViewStyle = {
     ...smartpriceModalStyles.activityIndicatorViewStyle,
     display: isBusy ? 'flex' : 'none',
+    position:
+      Platform.OS === 'android'
+        ? isBusy
+          ? 'absolute'
```
Comment (shubhagupta10) [inline src/modal/smartprice-modal.tsx:316]: does absolute work here for android (in smartprice-modal.styles.ts its commented)

Also can you create a new styles in the style file itself?
activityIndicatorViewStyleBusy,  activityIndicatorStyleAndorid,  activityIndicatorStyleAndoridBusy and use it accordingly here?

**Review by shubhagupta10 — COMMENTED** [review]
I have added some comments on individual file.
Apart from that
1) please add ignore for idea files
2) the "Phone number" placeholder is too close to border
3) Also check the exception [Unhandled promise rejection: TypeError: undefined is not an object (evaluating 'error.response.data.details.errors')]
Stack trace: src\api\smartprice-api.ts:160:43 in axios.post.then._catch$argument_0 node_modules\react-native\node_modules\promise\setimmediate\core.js:37:13

File: src/modal/smartprice-modal.tsx (hunk 3 above)
Comment (toledoal) [reply src/modal/smartprice-modal.tsx:316]: Correct, yes, makes sense

**Review by shubhagupta10 — APPROVED** [review]

**Comment by toledoal** [comment]
-Removed .idea folders.
-Phone number fixed
-Exception handled

**Pull request merged**